sentry_sdk/integrations/logging.py

```diff
@@ -29,7 +29,26 @@ class LoggingIntegration(Integration):
     identifier = "logging"
 
     def __init__(self, level=DEFAULT_LEVEL, event_level=DEFAULT_EVENT_LEVEL):
-        self._handler = SentryHandler(level=level, event_level=event_level)
+        self._handler = None
+        self._breadcrumb_handler = None
+
+        if level is not None:
+            self._breadcrumb_handler = BreadcrumbHandler(level=level)
+
+        if event_level is not None:
+            self._handler = SentryHandler(level=event_level)
+
+    def _handle_record(self, record):
+        if (
+            self._breadcrumb_handler is None
+            or record.levelno < self._breadcrumb_handler.level
+        ):
+            return
+
+        if self._handler is not None and record.levelno >= self._handler.level:
+            self._handler.handle(record)
+
+        self._breadcrumb_handler.handle(record)
```

Comment:
This can be `None`.

Reply:
No, because of line 43 (not sure about this interaction between event_level and level anymore, but I don't want to break that API now)

Reply:
I think the logic should be separated and it's fine "breaking" it.

Reply:
Done